### PR TITLE
buildApp fix for Issue 35

### LINKED ISSUE #35: buildApp should first check for function
It needs to check for function (AppClass) first and then check if it's just an object because:

`_.isObject(function(){}) == true`

### PR DESCRIPTION
**Pull request by TerrenceLJones**
Refactor code to check first to see if AppClass is a function. Also added tests. This PR fixes #35.

### CODE REVIEW

**Comment by TerrenceLJones** [comment]
Reference this PR originated from PR [#37](https://github.com/RoundingWellOS/marionette.toolkit/pull/37). That PR was request against the wrong branch.

File: test/unit/app-child-manager.spec.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import '../../src/marionette.toolkit';
+
+describe('App Manager', function() {
+  describe('when passing a class to buildApp', function() {
+    beforeEach(function() {
+      this.myApp = new Marionette.Toolkit.App();
+    });
+
+    it('should return and instance of the clas', function() {
```
Comment (brentli1) [inline test/unit/app-child-manager.spec.js]: 'should return an instance of the class'

**Comment by paulfalgout** [comment]
:+1: just waiting on travis

File: test/unit/app-child-manager.spec.js (hunk 1 above)
Comment (paulfalgout) [inline test/unit/app-child-manager.spec.js]: good catch

Comment (TerrenceLJones) [inline test/unit/app-child-manager.spec.js]: Yea I saw that. Thanks. About to fix it.

**Comment by brentli1** [comment]
:+1:

**Pull request merged**